src/functions-and-arrays.js

```diff
@@ -1,41 +1,90 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(num1, num2) {
+  return num1 > num2 ? num1 : num2;
```

Comment:
Great work using the ternary operator already! 👏🏻 👏🏻 

Another way to solve this one would be to use the `Math.max()` method. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)